src/libraries/System.Net.Http.WinHttpHandler/tests/UnitTests/APICallHistory.cs

```diff
@@ -70,6 +70,8 @@ public static ProxyInfo RequestProxySettings
 
         public static List<IntPtr> WinHttpOptionClientCertContext { get { return winHttpOptionClientCertContextList; } }
 
+        public static (uint KeepAliveTime, uint KeepAliveInterval)? TcpKeepaliveOptions { get; set; }
```

Comment:
Since the other properties here are named WinHttpOption*, should we do the same for this one?